experimental/trusted_shuffler/scripts/run_trusted_shuffler

```diff
@@ -1,8 +1,4 @@
 #!/usr/bin/env bash
 
-RUST_LOG=info cargo run --manifest-path=experimental/trusted_shuffler/server/Cargo.toml -- \
---k=1 --listen-address="[::]:8080" --backend-url="http://localhost:8888" 
-   
-# Convenient to know the PID to kill when the Trusted Shuffler crashes.
-echo "Trusted Shuffler PID: "
-echo $!
+RUST_LOG=debug cargo run --manifest-path=experimental/trusted_shuffler/server/Cargo.toml -- \
+--k=1 --listen-address="[::]:8080" --backend-url="http://localhost:8888"
```

Comment:
I think single quotes are sufficient, there is no interpolation going on here.

Reply:
done.